src/internal/sdata/sql.go

```diff
@@ -2,36 +2,82 @@ package sdata
 
 import (
 	"fmt"
+	"reflect"
 	"strings"
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
 )
 
+// limit the number of rows
```

Comment:
maybe mention this is specifically about batching here, though it's not a big file I guess